src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/NotifyServiceTest.kt

```diff
@@ -257,6 +260,30 @@ class NotifyServiceTest {
     verifyNoInteractions(notificationClient)
   }
 
+  @Test
+  fun `send unapproved licence email to probation practitioner`() {
+    val emailContent = listOf(
+      UnapprovedLicence(
+        "a123",
+        "jim",
+        "smith",
+        "comFirst",
+        "comLast",
+        EMAIL_ADDRESS
+      )
+    )
+
+    val expectedMap = mapOf(
+      Pair("crn", "a123"),
+      Pair("prisonerFirstName", "jim"),
+      Pair("prisonerLastName", "smith"),
+      Pair("comName", "comFirst comLast")
```

Comment:
rather than use Pair, its a bit more idiomatic to use the infix method `to`

e.g: 
```
mapOf(
      "crn" to "a123",
      "prisonerFirstName" to "jim",
)
```

Reply:
will do